assembly/example/activemq/activemqSub.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"context"
+	"github.com/andile-innovation/popcorn/log"
+	"github.com/andile-innovation/stream/activemq"
+	"os"
+	"os/signal"
+	"strconv"
+	"sync"
+	"syscall"
+	"time"
+)
+
+func main() {
+
+	amqSub := activemq.NewAMQPSubscriber(activemq.NewAMQPSubscriberRequest{
+		Host: "localhost",
+		Port: 5672,
+		Username: "admin",
+		Password: "admin",
+	})
+
+	if err := amqSub.Connect(); err != nil {
+		panic(err)
+	}
+	messageReceivedC := make(chan string)
+	amqSub.Subscribe("MYQ1", messageReceivedC)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	gracefulStop := make(chan os.Signal, 2)
+	signal.Notify(gracefulStop, os.Interrupt)
+	signal.Notify(gracefulStop, os.Interrupt, syscall.SIGINT)
```

Comment:
you don't need two calls to Notify - only one with all the signals you want to catch